src/rgw/rgw_sal_motr.cc

```diff
@@ -721,6 +721,9 @@ int MotrBucket::put_info(const DoutPrefixProvider *dpp, bool exclusive, ceph::re
   bufferlist bl;
   struct MotrBucketInfo mbinfo;
   string tenant_bkt_name = get_bucket_name(info.bucket.tenant, info.bucket.name);
+  
+  // Set exclusive to false for put bucket tags
+  exclusive = (attrs.find(RGW_ATTR_TAGS) == attrs.end());
```

Comment:
This change should not affect any other API, except PutBucketTagging, please test enough.
Does this come in any Obj APIs too where Tagging are being added for an Object.

Reply:
1. API's with known issues are not blocked now and works fine.
2. No merge_and_store_attrs is not used by any Obj Tagging APIs

Reply:
Ok fine.